### DISCUSSION TITLE
Wiki

### DISCUSSION THREAD

**Discussion by richardbirkin** [SuperHyperInstantFutureTime/TrackShift#3] [General]
Would a wiki be helpful?

Pages that come to mind:

- TrackShift mission statement
- Music Royalties terminology/definitions

Chuck suggestions in the comments at any point and I will wiki-serve you.

**Comment by g105b** [comment]
Yeah, I think so. The nature of wikis being flat files makes sense for this kind of thinking. This discussions section is great for ... well, discussions. But as soon as the topics of conversation can are locked in we can use it to document everything as a knowledge base.

A glossary of music royalties would be really useful because even though we've spoken a lot about this topic, you often come out with a word that doesn't quite make sense to me, or I assume means something completely different.

A mission statement will be good to see on the wiki's homepage. Something for us to reflect on, but also the users and for people in the music industry to read and instantly understand our vibe.

**Reply by richardbirkin** [reply]
Great. I've been looking at the _good examples of how to use Github's wiki_ articles and get it now. Will press on with that next.
